<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainW</class>
 <widget class="QMainWindow" name="MainW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainW</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="XOGrView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>482</width>
      <height>482</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>500</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>125</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>220</y>
      <width>231</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lab_turn">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>370</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Restart the game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_comp">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Computer's move</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>787</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
